system/Images/Handlers/GDHandler.php

```diff
@@ -267,7 +267,7 @@ public function save(?string $target = null, int $quality = 90): bool
                     throw ImageException::forInvalidImageCreate(lang('Images.webpNotSupported'));
                 }
 
-                if (! @imagewebp($this->resource, $target)) {
+                if (! @imagewebp($this->resource, $target, $quality)) {
```

Comment:
This PR changes the default quality from -1 to 90.
https://www.php.net/manual/en/function.imagewebp.php#refsect1-function.imagewebp-description
I don't know what -1 means.
Does this degrade the image quality?

Reply:
It's probably the same as in `imagejpeg()` - https://www.php.net/manual/en/function.imagejpeg.php

> The default (-1) uses the default IJG quality value (about 75).

Reply:
80?
https://github.com/php/php-src/blob/8d0345d94ef98bfe29df264b338aeef16ddefda9/ext/gd/libgd/gd_webp.c#L121

Reply:
Yes, it appears to be a fixed value.

Reply:
Since the behavior changes (80 -> 90), should we add this to changelog?
This doesn't seem to destroy the application, though.

Reply:
Well... it's a change in behavior, so we can mention it.

Reply:
I think we must change default $quality to 80, And i can set parameter $quality to PNG with 0 - 9.

Reply:
Do you mean that the default quality for webp is 80, for jpeg is 90?

Reply:
I think we should stay with 90 as the default value for quality. Having a separate value for WEBP makes not much sense to me.

Regarding PNG, we can't speak about the quality since these files are lossless. The image quality, as we know it from JPEG or WEBP, will not be changed. Only the compression - file size will be changed.

Reply:
Its okay, i agree to `$quality` to `WebP` using default 90. And with this changes make developer have preferences to changes quality compression for WebP

Reply:
@ddevsr This changes the default quality for webp from 80 to 90.
Can you add it in https://github.com/codeigniter4/CodeIgniter4/blob/4.4/user_guide_src/source/changelogs/v4.4.0.rst#changes ?